modules/end-user-guide/pages/developer-workspaces.adoc

```diff
@@ -0,0 +1,17 @@
+:navtitle: Using developer workspaces
+:keywords: user-guide, workspaces-overview
+:page-aliases: .:using-developer-workspaces, .:workspaces-overview, .:what-are-workspaces.html, .:creating-and-configuring-a-new-workspace, creating-and-configuring-a-new-workspace, authenticating-on-scm-server-with-a-personal-access-token, navigating-che, navigating-che-using-the-dashboard, importing-certificates-to-browsers, creating-a-workspace-from-local-devfile-using-chectl, importing-kubernetes-applications-into-a-workspace, workspaces-overview, configuring-a-workspace-with-dashboard, creating-a-workspace-from-a-code-sample, creating-a-workspace-from-a-template-devfile, creating-a-workspace-from-remote-devfile, running-a-workspace-with-dashboard, remotely-accessing-workspaces, importing-the-source-code-of-a-project-into-a-workspace
```

Comment:
```suggestion
:page-aliases: .:using-developer-workspaces, .:workspaces-overview, .:what-are-workspaces.html, .:creating-and-configuring-a-new-workspace, creating-and-configuring-a-new-workspace, authenticating-on-scm-server-with-a-personal-access-token, navigating-che, navigating-che-using-the-dashboard, importing-certificates-to-browsers, creating-a-workspace-from-local-devfile-using-chectl, importing-kubernetes-applications-into-a-workspace, workspaces-overview, configuring-a-workspace-with-dashboard, creating-a-workspace-from-a-code-sample, creating-a-workspace-from-a-template-devfile, creating-a-workspace-from-remote-devfile, running-a-workspace-with-dashboard, remotely-accessing-workspaces, importing-the-source-code-of-a-project-into-a-workspace, using-developer-workspaces
```

Reply:
Thank you, Fabrice.
I've applied this.
Apologies for the nuisance: next time, I'll be more careful.